### DISCUSSION TITLE
Error de animación de clic utilizando la librería customtkinter

### DISCUSSION THREAD

**Discussion by feerxz** [Akascape/CTkTable#52] [Q&A]
Hola comunidad de Github, creé una pequeña y simple app de escritorio en Python, utilizando las librerías de interfaces gráficas Tkinter y CustomTkinter, al parecer todo funciona bien pero tengo un pequeño error de animación de clic al hacer clic en un widget de botón para cerrar sesión, les dejaré el script de Python y unas capturas de cómo sale el error para que si pueden, me ayuden por favor. 
Pdta.: Soy nuevo en la programación y recién estoy aprendiendo interfaces gráficas así que discúlpenme el código spaghetti, esto fue mi proyecto de fin de semestre para la universidad. 

@Akascape   no se si puedas revisar mi código fuente y ayudarme por favor, utilicé 2 librerías tuyas, te lo agradecería mucho.

[INTERFAZ_BANCO.zip](https://github.com/Akascape/CTkTable/files/12900452/INTERFAZ_BANCO.zip) ----->>Aquí está el archivo .py comprimido

![Captura](https://github.com/Akascape/CTkTable/assets/139174928/baf50568-d5b4-4f4a-8fdf-828ac519a000)---->> Luego de hacer clic en el botón "Cerrar Sesión" que se indica en la imagen, funciona bien pero en la terminal me sale este error de aquí abajo:

![Captura2](https://github.com/Akascape/CTkTable/assets/139174928/b7c87444-d383-43dc-a916-2ca3fbf5e73c)

**Comment by Akascape** [comment]
@feerxz Este problema está relacionado con customtkinter. Por favor, envíe el problema en https://github.com/TomSchimansky/CustomTkinter/discussions
